core/ir/instr_api.h

```diff
@@ -298,10 +298,14 @@ struct _instr_t {
 
     uint opcode;
 
-#    ifdef X86
-    /* PR 251479: offset into instr's raw bytes of rip-relative 4-byte displacement */
-    byte rip_rel_pos;
-#    endif
+    union {
+        /* PR 251479: offset into instr's raw bytes of rip-relative 4-byte displacement */
```

Comment:
Explain that this is used only for x86.  Maybe the ifdef should be left around this; if it's not left, it is weird to have all the code touching it inside ifdefs.

Reply:
Done, re-introduced ifdef.